packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.test.tsx

```diff
@@ -111,30 +108,21 @@ describe('PageAccountRecoveryConfirmKey', () => {
 
   it('renders the component as expected when provided with an expired link', async () => {
     const accountWithTokenError = {
+      resetPasswordStatus: jest.fn().mockResolvedValue(false),
       verifyPasswordForgotToken: jest.fn().mockImplementation(() => {
         throw AuthUiErrors.INVALID_TOKEN;
       }),
     } as unknown as Account;
-    render(<Subject {...{ account: accountWithTokenError }} />);
-
-    await typeByLabelText('Enter account recovery key')(MOCK_RECOVERY_KEY);
-    fireEvent.click(
-      screen.getByRole('button', { name: 'Confirm account recovery key' })
-    );
+    renderSubject(accountWithTokenError);
 
     await screen.findByRole('heading', {
       name: 'Reset password link expired',
     });
-    screen.getByText('The link you clicked to reset your password is expired.');
-    screen.getByRole('button', {
-      name: 'Receive new link',
-    });
   });
 
   describe('renders the component as expected when provided with a damaged link', () => {
     it('with missing token', async () => {
-      mockToken = null;
-      render(<Subject {...{ account }} />);
+      renderSubject(undefined, paramsWithMissingToken);
```

Comment:
This is fine to leave as-is, but for future reference or if this grows to be bigger, you could also change this to an options object argument that's destructured in `renderSubject` and you'd call it like `renderSubject({params: paramsWithMissingToken})`.